### ISSUE TITLE
MDTimePicker cannot be instantiated

### ISSUE THREAD

**Issue by x4dr** [kivymd/KivyMD#1308]
Code

```
from kivymd.app import MDApp
from kivymd.uix.pickers import MDTimePicker

class Test(MDApp):
    def build(self):
        return MDTimePicker()


if __name__ == "__main__":
    Test().run()

```
causes 

```
   File "/home/maric/PycharmProjects/remind/spielwiese.py", line 10, in <module>
     Test().run()
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/app.py", line 954, in run
     self._run_prepare()
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/app.py", line 924, in _run_prepare
     root = self.build()
   File "/home/maric/PycharmProjects/remind/spielwiese.py", line 6, in build
     return MDTimePicker()
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/pickers/timepicker/timepicker.py", line 567, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/pickers/datepicker/datepicker.py", line 470, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/theming.py", line 1509, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/modalview.py", line 195, in __init__
     super(ModalView, self).__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/anchorlayout.py", line 68, in __init__
     super(AnchorLayout, self).__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/layout.py", line 76, in __init__
     super(Layout, self).__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/behaviors/elevation.py", line 1344, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/behaviors/backgroundcolor_behavior.py", line 206, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/behaviors/backgroundcolor_behavior.py", line 162, in __init__
     super().__init__(**kwarg)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/behaviors/elevation.py", line 851, in __init__
     super().__init__(**kwargs)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/widget.py", line 366, in __init__
     self.apply_class_lang_rules(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules
     Builder.apply(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/lang/builder.py", line 540, in apply
     self._apply_rule(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/lang/builder.py", line 662, in _apply_rule
     self._apply_rule(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/lang/builder.py", line 660, in _apply_rule
     child.apply_class_lang_rules(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules
     Builder.apply(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/lang/builder.py", line 540, in apply
     self._apply_rule(
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivy/lang/builder.py", line 658, in _apply_rule
     child = cls(__no_builder=True)
   File "/home/maric/PycharmProjects/remind/venv/lib/python3.10/site-packages/kivymd/uix/pickers/timepicker/timepicker.py", line 201, in __init__
     self.bind(text_color=self.setter("hint_text_color_normal"))
   File "kivy/_event.pyx", line 452, in kivy._event.EventDispatcher.bind
 KeyError: 'text_color'

Process finished with exit code 1

```

### Versions

* OS: linux (5.18.16-arch1-1)
* Python:  v3.10.5 (main, Aug  1 2022, 07:53:20) [GCC 12.1.0]
* Kivy: v2.1.0
* KivyMD: 1.1.0.dev0, git-Unknown, 2022-08-09 (installed with the method from the readme, previously it was 1.0.1, which behaved the same)

I assume it is something i am doing, however this error message is not helpful
providing `text_color="#000000"` or similar things does not help either

**Comment by HeaTTheatR** [comment]
https://kivymd.readthedocs.io/en/1.0.1/components/datepicker/

**Comment by x4dr** [comment]
> https://kivymd.readthedocs.io/en/1.0.1/components/datepicker/
there seems to be a lack of explanation here? is that a "use this instead?"

I am using the first provided example from https://kivymd.readthedocs.io/en/1.0.1/components/timepicker/
(not date picker)
since i was asked for a minimal code example, i minified it even more so there is no button to be pushed to cause the crash, it just crashes on startup.
The first code example there also reproduces it (in a fresh venv on my system)

I have tried to find something in the datepicker documentation that would help me or explain and I have failed, I also cannot ask on discord, as you dont allow users that did not allow discord access to their phone numbers to post there.

Unless you mean the `The widget is under testing. Therefore, we would be grateful if you would let us know about the bugs found.` ... but then this is what i am trying to do here
